packages/cli/src/commands/setup/deploy/providers/render.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from 'src/lib'
+
+const RENDER_YAML = `services:
+- type: web
+  name: redwood-api
+  env: node
+  plan: starter
+  buildCommand: yarn && yarn rw prisma migrate deploy && yarn rw build api
+  startCommand: cd api && yarn api-server --functions ./dist/functions
+  envVars:
+  - key: DATABASE_URL
+    fromDatabase:
+      name: redwood-db
```

Comment:
Would there be an issue or collision if one deploys multiple apps to render if the same `redwood-db` is used on a single Render "account" -- or is the name exclusive to this deploy?

Should the user be given the option to name the database? Or derive it from the repo/project name?

I assume this is the "Friendly Name" as set when manually creating a database?

Reply:
Hey, great question. If a user created a second instance of redwood-db as part of an additional project. They would get a message asking if they want to recreate the already deployed services or create new ones.

If they decided to make new ones then the `redwood-db` name would get an identifier appended to it for that service group. It would then look like this.
![image](https://user-images.githubusercontent.com/12256202/112683457-f6d66500-8e47-11eb-8d97-350c8d02bbf4.png)

That being said, it might still be a better user experience to name the services based on the name of their project instead of a generic "redwood". If the user wanted to rename their services beyond that it would probably be easiest for them just to modify the generated `render.yaml`.

Reply:
I was mostly concerned if the deploy would fail.

If the db friendly name is set with `an identifier appended to it` I would imagine that I'd want to modify my `render.yaml` to still be in sync?

Reply:
Render automatically links the right database even when we add an identifier to the friendly name.

Reply:
resolved in latest per discussion